extension/src/popup/SorobanContext.tsx

```diff
@@ -9,9 +9,11 @@ import {
 
 import { settingsNetworkDetailsSelector } from "./ducks/settings";
 
+const BASE_FEE = "100";
```

Comment:
you should use `SorobanClient.BASE_FEE`, I think

Reply:
thanks, I keep meaning to update that. Done in 05fa8fe00e388f8cbf24ba50b6f75548e8ba489d